src/plugins/cosmos/components/modals/Staking/views/ClaimConfirm.tsx

```diff
@@ -34,16 +46,15 @@ export type StakingValues = {
 
 type ClaimConfirmProps = {
   assetId: CAIP19
-  cryptoStakeAmount: BigNumber
-  fiatAmountAvailable: string
+  accountSpecifier: CAIP10
```

Comment:
Not always a CAIP10 for all accounts (e.g UTXOs) but in the context of Cosmos, it actually is